alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/opt/buildkite
+export GEM_HOME=${HOME}/.gem
+
+cd $HOME
```

Comment:
>  default the working directory for any process ... should be changed to /.

I agree with this. You could have a section in your Dockerfile that undoes the things setup in the parent Dockerfile.

Later on we can move things around so that you don't have to override so much.